xml_stream/__init__.py

```diff
@@ -13,7 +13,7 @@ def _convert_xml_element_to_dict(xml_element: Element) -> XmlDictElement:
     return XmlDictElement(xml_element)
 
 
-def read_xml_file(file_path: str, records_tag: Optional[str], to_dict: Optional[bool] = False,
+def read_xml_file(file_path: str, records_tag: Optional[list], to_dict: Optional[bool] = False,
                   **kwargs) -> Union[Iterator[Element], Iterator[XmlDictElement]]:
```

Comment:
here the return should say tuple of element, str? same applies to `read_xml_string`

Reply:
Done!